script.js

```diff
@@ -0,0 +1,200 @@
+let webBrands = [];
+let brands=[];
+let resultbrands=[];
+let productNode;
+const proxyurl = "https://cors-anywhere.herokuapp.com/";
+const url = `https://wellspent-ethical.herokuapp.com/api/brands`;
+
+
+var linksList = document.querySelectorAll("[id^=product-]");
+linksList.forEach(function(product) {
+  const node = product.childNodes[0];
+  const label = node.getAttribute("aria-label");
+  let brand = label.toLowerCase();
+  webBrands.push(brand);
```

Comment:
This could be done in one line instead of declaring another variable.
``` 
const label = node.getAttribute("aria-label").toLowerCase();
webBrands.push(label); 
```